CASC/PortfolioMode.cpp

```diff
@@ -513,6 +577,14 @@ void PortfolioMode::runSlice(vstring sliceCode, int timeLimitInDeciseconds)
   try
   {
     Options opt = *env.options;
+
+    // opt.randomSeed() would normally be inherited from the parent
+    // addCommentSignForSZS(cout) << "runSlice - seed before setting: " << opt.randomSeed() << endl;    
+    if (env.options->randomizedPortfolioWorkers()) {
+      // but here we want each worker to have their own seed
+      opt.setRandomSeed(getpid());
```

Comment:
This seems dubious as PIDs might be recycled. Maybe `std::random_device` again, or a combination of PID and UNIX time?

Reply:
A good point.